packages/web-workers/scan-batch-requests-feed-func/function.json

```diff
@@ -9,7 +9,8 @@
             "databaseName": "onDemandScanner",
             "collectionName": "scanBatchRequests",
             "createLeaseCollectionIfNotExists": true,
-            "startFromBeginning": true
+            "startFromBeginning": true,
```

Comment:
do we still need startFromBeginning? what happens if there are multiple function instances? will it make sure it doesnt send duplicate records?

Reply:
I will keep startFromBeginning It will not break but allow to re-process by trigger